admin/js/getting-started.js

```diff
@@ -0,0 +1,24 @@
+(function($){
```

Comment:
and looks like it's better to use document ready event, e.g. `jQuery(function($) { rest of the code })`